### PR TITLE
(DOCSP-18921): Add backlink to the glossary

### PR DESCRIPTION
**Pull request by nspektor**
### Jira

- https://jira.mongodb.org/browse/DOCSP-18921

### Staged Changes (Requires MongoDB Corp SSO)

- [Glossary](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-18921/get-started/glossary/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/get-started/glossary.txt [hunk 1]
```diff
@@ -443,7 +443,21 @@ Glossary
    relationship
      A relationship is an object property that references another object rather
      than a scalar or embedded object.
+
+   backlink
+      a backlink is a pointer that indicates that a RealmObject has an
```
Comment (MongoCaleb) [inline source/get-started/glossary.txt:448]: Capitol "A" needed.

Comment (MongoCaleb) [reply source/get-started/glossary.txt:448]: Is "pointer" the right word? I wonder if that will be confused with C++ pointers? Maybe "link" is better?

**Review by MongoCaleb — APPROVED** [review]
One nit and one possible wording change. LGTM otherwise.

File: source/get-started/glossary.txt (hunk 1 above)
Comment (nspektor) [reply source/get-started/glossary.txt:448]: yeah i was thinking the same thing but didnt think of "link", i like it!

**Pull request merged**